packages/components/organisms/f-form/src/components/_tests/Form.test.js

```diff
@@ -76,17 +96,205 @@ describe('Form', () => {
             });
         });
 
-        describe('onFormSubmit :: ', () => {
-            it('should emit `form-submitting`', () => {
+        describe('isFormValid ::', () => {
+            let touchSpy;
+
+            beforeEach(() => {
+                touchSpy = jest.spyOn($v, '$touch');
+            });
+
+            afterEach(() => {
+                jest.clearAllMocks();
+            });
+
+            it('should call `.touch()` ', () => {
```

Comment:
Do we need to specifically know that it has called `touch`, or is it better to check that the fields are now dirty?

Reply:
Having thought about this some more, I'm not sure how easy this will be whilst mocking the `$v` object without just testing your mock. I think what you've got now is fine